src/search/search.css

```diff
@@ -0,0 +1,48 @@
+.search_wrapper {
+    margin-top: 100px;
+    display: flex;
+    justify-content: center;
+}
+
+.search_input {
+    width: 700px;
+    height: 48px;
+    border: none;
+    background-color: #333333;;
+    border-radius: 5px;
+    padding-left: 20px;
+    outline: none;
```

Comment:
[Here](https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/) you can find interesting info about ordering of properties in css rule